lib/ssh/src/ssh_acceptor.erl

```diff
@@ -164,17 +164,18 @@ handle_connection(_Address, _Port, {error,Error}, _Options, _Socket, _MaxSession
 
 handle_connection(Address, Port, _Peer, Options, Socket, _MaxSessions, _NumSessions, ParallelLogin)
   when ParallelLogin == false ->
-    handle_connection(Address, Port, Options, Socket);
+    handle_connection(undefined, Address, Port, Options, Socket);
 
 handle_connection(Address, Port, _Peer, Options, Socket, _MaxSessions, _NumSessions, ParallelLogin)
   when ParallelLogin == true ->
+    Parent = self(),
     Ref = make_ref(),
     Pid = spawn_link(
```

Comment:
Spawning this process here that is not part of the supervisor tree  and linking it to the user process I think is problematic.